samples/features/sql-management-objects/README.md

```diff
@@ -0,0 +1,62 @@
+# SmoSamples
+
+This unit test project is meant to demonstrate features of the Sql Management Objects framework and to help developers optimize performance of their SMO-based applications.
+
+
+### Contents
+
+[About this sample](#about-this-sample)<br/>
```

Comment:
nit: The <br />'s are unnecessary

Reply:
can you elaborate? I don't see any 's

Reply:
haha, whoops.

&lt;br /&gt;'s is what I meant.

Reply:
oh it's from the template